open-policy-agent/authorization/prevent_default_serviceaccount_usage/test/not_approved_pod_2.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
@tommeramber 
This example is not good because the policy checks that serviceAccountName != default